LucasTerra.lua

```diff
@@ -139,24 +139,24 @@ local closeddoorsids = {1628, 1629, 1631, 1632, 1638, 1640, 1642, 1644, 1646, 16
 local openeddoorsids = {1630, 1633, 1639, 1641, 1643, 1645, 1647, 1649, 1652, 1655, 1657, 1659, 1661, 1663, 1665, 1667, 1670, 1673, 1675, 1677, 1679, 1681, 1684, 1686, 1688, 1690, 1693, 1695, 1697, 1699, 2178, 2180, 4911, 4914, 5083, 5085, 5099, 5101, 5103, 5105, 5108, 5110, 5112, 5114, 5117, 5117, 5119, 5121, 5123, 5126, 5128, 5130, 5132, 5135, 5138, 5141, 5144, 5279, 5282, 5284, 5286, 5288, 5290, 5292, 5294, 5515, 5734, 5737, 5746, 5748, 6193, 6196, 6198, 6200, 6202, 6204, 6206, 6208, 6250, 6253, 6255, 6257, 6259, 6261, 6263, 6265, 6893, 6895, 6897, 6899, 6902, 6904, 6906, 6908, 7035, 7037, 7039, 7041, 7044, 7046, 7048, 7050, 7713, 7716, 7718, 7720, 7722, 7724, 7726, 7728, 7869, 8251, 8254, 8256, 8258, 8260, 8262, 8264, 8266, 8353, 8356, 8358, 8360, 8362, 8364, 8366, 8368, 9353, 9356, 9358, 9360, 9362, 9364, 9366, 9368, 9553, 9555, 9557, 9559, 9562, 9564, 9566, 9568, 9860, 9864, 9866, 9869, 9873, 9875, 11234, 11238, 11240, 11243, 11247, 11249, 11708, 11716, 17562, 17564, 17566, 17568, 17571, 17573, 17575, 17577, 17702, 17704, 17706, 17708, 17711, 17713, 17715, 17717, 20445, 20447, 20449, 20451, 20454, 20456, 20458, 20460}
 
 local cityAreas = {
-   {"thais", 32369, 32241, 100},
-   {"carlin", 32343, 31791, 60},
-   {"kazordoon", 32629, 31925, 60},
-   {"kazordoon", 32826, 31762, 15},
-   {"ab'dendriel", 32681, 31637, 70},
-   {"edron", 33205, 31819, 60},
-   {"darashia", 33238, 32435, 60},
-   {"venore", 32957, 32076, 100},
-   {"ankrahmun", 33158, 32829, 100},
-   {"port hope", 32623, 32763, 60},
-   {"liberty bay", 32317, 32826, 80},
-   {"svargrond", 32273, 31149, 80},
-   {"yalahar", 32802, 31206, 80},
-   {"travora", 32067, 32354, 20},
-   {"farmine", 33023, 31453, 60},
-   {"gray beach", 33447, 31323, 30},
-   {"roshamuul", 33553, 32379, 50},
-   {"rathleton", 33627, 31913, 50}
+   {1, "thais", 32369, 32241, 100},
+   {2, "carlin", 32343, 31791, 60},
+   {3, "kazordoon", 32629, 31925, 60},
+   {3, "kazordoon", 32826, 31762, 15},
+   {4, "ab'dendriel", 32681, 31637, 70},
+   {5, "edron", 33205, 31819, 60},
+   {6, "darashia", 33238, 32435, 60},
+   {7, "venore", 32957, 32076, 100},
+   {8, "ankrahmun", 33158, 32829, 100},
+   {9, "port hope", 32623, 32763, 60},
+   {10, "liberty bay", 32317, 32826, 80},
+   {11, "svargrond", 32273, 31149, 80},
+   {12, "yalahar", 32802, 31206, 80},
+   {13, "travora", 32067, 32354, 20},
+   {14, "farmine", 33023, 31453, 60},
+   {15, "gray beach", 33447, 31323, 30},
+   {16, "roshamuul", 33553, 32379, 50},
+   {17, "rathleton", 33627, 31913, 50}

```

Comment:
It could be useful for detecting city, which could easier depositing stuff for someone.
Lucas also said he will be using these coords so it might be useful for the future stuff.